aries-core/src/js/components/helpers/Anchors/NavLink.js

```diff
@@ -18,7 +18,7 @@ export const AnchorNavLink = ({ ...rest }) => {
             },
             textDecoration: 'none',
```

Comment:
Do we still need this `textDecoration: 'none'` even though it's in the theme now?

Reply:
This one is at the `hover` level. The textDecoration on the theme is for normal state.

Reply:
Gotcha! Makes sense then, I missed that.

Reply:
I think this hover behavior should also be part of the aries-site theme rather then being placed on a ThemeContext component.

Reply:
That will apply to all instances of anchor. Is that the desired behavior? For instance, the AnchorCallToAction links, what are their hover states?